src/model.js

```diff
@@ -28,19 +43,19 @@ export default class MediaPlayerObject {
   }
 
   get isIdle() {
-    return this.state === 'idle';
+    return this.states.idle.indexOf(this.state) >= 0;
```

Comment:
```suggestion
    return this.states.idle.includes(this.state);
```
Is more clear.
Same for all state checks.